src/models/session/directorToSign.model.ts

```diff
@@ -1,6 +1,7 @@
 export default interface DirectorToSign {
   id: string
   name: string
+  isApplicant: boolean
```

Comment:
this was introduced to differentiate between the director selected on the selectDirector page vs the signatories selected on the selectSignatories page. Only the directors selected on the selectSignatories page will be shown on the define signatory info page.